go/mcap/reader_test.go

```diff
@@ -394,150 +394,49 @@ func TestReaderCounting(t *testing.T) {
 	}
 }
 
-func TestMCAPInfo(t *testing.T) {
+func TestReaderMetadataCallback(t *testing.T) {
 	cases := []struct {
-		assertion   string
-		schemas     []*Schema
-		channels    []*Channel
-		messages    []*Message
-		metadata    []*Metadata
-		attachments []*Attachment
+		assertion string
+		useIndex  bool
 	}{
 		{
-			"no metadata or attachments",
```

Comment:
Hard to tell what's being removed in this test — I assume this is unneeded for metadata testing, but are we losing coverage for something else (since this test used to be called something else?)

Reply:
weird - I wasn't exepecting anything removed